src/pypgstac/pyproject.toml

```diff
@@ -20,14 +20,15 @@ classifiers = [
     "Programming Language :: Python :: 3.11",
 ]
 dependencies = [
-    "smart-open>=4.2",
-    "orjson>=3.5.2",
-    "python-dateutil==2.8.*",
+    "cachetools==5.3.*",
     "fire==0.4.*",
+    "hydraters==0.1.0",
```

Comment:
what do you think about `==0.1.*` just in case we find a bug we need to fix with a patch release